samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```diff
@@ -31,6 +31,8 @@
 
     <!-- Uncomment the line below if Android App installation fails -->
     <!--<EmbedAssembliesIntoApk>true</EmbedAssembliesIntoApk>-->
+
+    <WindowsSdkPackageVersion>10.0.19041.41</WindowsSdkPackageVersion>
```

Comment:
Why do we need to do this when the MVVM toolkit project refers to an older package: https://github.com/CommunityToolkit/dotnet/blob/main/src/CommunityToolkit.Mvvm/CommunityToolkit.Mvvm.csproj

@Sergio0694 do you know which value is correct?

Reply:
.38 is the minimum to build, but if you're manually setting the package version you should set the latest available at that time that's compatible. It was .41 when I made the last MVVM Toolkit update. You can use .53 today. Also FYI, once you get the updated .NET SDK, you should be able to remove that property, as the .NET SDK should set the correct Windows SDK package version automatically 🙂

Reply:
@Sergio0694 thank you for the speedy response! When should we get the updated .NET SDK? What version of the SDK are you referring to there?

Reply:
I think the new .NET SDK that went out in the October patch Tuesday should have it (8.0.403) 🤔

Reply:
Thanks Sergio! I've updated it to `10.0.19041.53`.